sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/models/ContentType.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.generic.core.http.models;
+
+/**
+ * The different values that commonly used for Content-Type header.
+ */
+public final class ContentType {
```

Comment:
We removed this at some point and I think I ended up adding it back as implementation class, we should consider keeping it there if it's not going to be used outside of this package. We can make it public when we need it.

Reply:
yep don't need this/